src/components/Wizard/WizardItem/WizardItemPoint/index.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { WizardItemPoint } from '.'
+
+const meta: Meta<typeof WizardItemPoint> = {
+  component: WizardItemPoint,
+  tags: ['autodocs'],
+  argTypes: {
+    variant: {
+      control: { type: 'select' },
+      options: ['ACTIVE', 'INACTIVE', 'ERROR'],
+      description:
+        'Available variants "active", "inactive" and "error". Choose the most suitable one according to progress status. Default value "inactive".',
+    },
+  },
+}
+
+const checkIcon = (color = 'white') => (
```

Comment:
check if we already have this icon. If not, ask the UX team to put it in figma

Reply:
This icon is being used for illustrative purposes only and is not part of the component. In a real case, a library icon may or may not be used.